crates/sui-transaction-checks/src/lib.rs

```diff
@@ -162,13 +162,6 @@ mod checked {
         _receiving_objects: ReceivingObjects,
     ) -> SuiResult<CheckedInputObjects> {
         kind.validity_check(config)?;
-        if kind.is_system_tx() {
```

Comment:
Why did we remove this check here? We probably still don't want to dev inspect a system transaction kind?

Reply:
It seems unnecessary? There isn't any harm in it is there? I can put it back if you disagree.